src/lib/components/CartoonCard.svelte

```diff
@@ -0,0 +1,13 @@
+<div class="w-96 rounded-b-[50px] rounded-t-[180px] bg-csi-blue px-2 pt-20">
+    <img src="src/lib/cartoon-card-placeholder.svg" alt="Corporate Cartoon" />
+</div>
+
+<div class="w-96">
+    <h1 class="font-dm text-3xl">Develop well-rounded DCS students</h1>
+    <p>with fundamental classroom learning through practical experience</p>
+
+    <!-- Attribution for Placeholder Image -->
+    <div class="mt-10 text-blue underline">
+        <a href="https://storyset.com/web">Web illustrations by Storyset</a>
+    </div>
```

Comment:
We can actually put this disclaimer in the footer part of the page. I believe there should be a `Footer` component by now thanks to #7.

Reply:
Removed na po! Should I add the attribution po in this same pull request once the footer component is merged or should I open a new pull request for that po later on?

Reply:
Yup, the footer component has been merged!

1. Head over to your repository page and **Sync Fork** your `main` branch.
2. From your local terminal, run `git switch main` and then `git pull` the latest stuff.
3. Then `git switch <feature>` and `git merge main` to bring the changes from `main` into your feature branch.

Unfortunately, though, you kinda branched out of `main` directly, which makes it extra difficult to merge the upstream `main` into your forked `main`. The easiest solution is just to re-fork the repo, unfortunately, and replay your changes.

> [!TIP]
> Don't forget to create a feature branch so that it's easier to sync the `main` branch in the future!